pkg/helm/controller/reconcile.go

```diff
@@ -240,7 +240,8 @@ func (r HelmOperatorReconciler) Reconcile(request reconcile.Request) (reconcile.
 			r.EventRecorder.Eventf(o, "Warning", "OverrideValuesInUse",
 				"Chart value %q overridden to %q by operator's watches.yaml", k, v)
 		}
-		previousRelease, updatedRelease, err := manager.UpdateRelease(context.TODO())
+		force := o.GetAnnotations()["helm.sdk.operator-framework.io/force"] == "true"
```

Comment:
One more tweak here. Let's call it `helm.sdk.operator-framework.io/upgrade-force`. The `Force` option is only applicable to the Upgrade action, but there are other options that are applicable to multiple actions that we would want separate annotations for, I think. For example, if we want to add support for `DisableHooks` in the future, we would want:
- ` helm.sdk.operator-framework.io/install-disable-hooks`
- ` helm.sdk.operator-framework.io/upgrade-disable-hooks`
- ` helm.sdk.operator-framework.io/uninstall-disable-hooks`